com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/message/Messages.java

```diff
@@ -66,6 +66,7 @@
     public static final String ILLEGAL_SERVICE_OPERATION_TYPE = "Illegal service operation type: {0}";
     public static final String MULTIPLE_CONFIGURATION_ENTRIES = "Multiple configuration entries were found matching the filter specified in resource \"{0}\": \"{1}\"";
     public static final String NO_CONTENT_TO_DEPLOY = "No content to deploy";
+    public static final String NO_SERVICE_PLAN_FOUND = "Could not create service instance {0}. Service plan {1} for service {2} not found";

```

Comment:
Where is this used?

Reply:
It was missed to be used. It will be used in CloudServiceOperator::findPlanForService()